.gitignore

```diff
@@ -1,12 +1,15 @@
-target/
-
-.DS_Store
-
-idea/
-/.idea
-consumer-example.iml
-/.classpath
-/.project
-/.settings/*
-/bin
-/*.log
+target/
+
+.DS_Store
+
+idea/
+/.idea
+consumer-example.iml
+/.classpath
+/.project
+/.settings/*
+/bin
+/*.log
+
+.env
+connect_start.sh
```

Comment:
Do not see this connect_start.sh.

Reply:
I am planning on re-working this, it will be updated after I can get the changes merged into the CDOT fork.